rsa/key.py

```diff
@@ -42,6 +43,8 @@
 import rsa.randnum
 import rsa.core
 
+warnings.simplefilter('always')
```

Comment:
I don't think this is something a library should do, or is it? I don't have much experience with the warnings module, but this looks like changing global configuration. If that's indeed the case, we shouldn't (it should be left to the application using our library).